types/json-file-plus/index.d.ts

```diff
@@ -9,17 +9,18 @@ declare namespace readJSON {
   function sync(filename: string): JSONFile;
 
   interface Format {
-    readonly indent: number;
+    readonly indent: number | '\t';
     readonly trailing: boolean;
   }
 
   class JSONData {
     format: Format;
     data: any;
     constructor(raw: string);
-    get: (key: string) => Promise<any>;
+    get: (key: PropertyKey) => Promise<any>;
     set: (data: any) => void;
-    remove: (key: string) => Promise<void>;
+    remove: (key: PropertyKey) => Promise<void>;
+    stringify: () => Buffer;
```

Comment:
how do i include the node types here?

Reply:
Add this to the top of the file:
```ts
/// <reference types="node" />
```
This indicates to consumers that `node` is a types dependency, since the `tsconfig.json` only applies to *this* project, not the consuming project.